content/knowledge-base/install-unity-version.md

```diff
@@ -38,13 +40,28 @@ Even though you are installing a different version of Unity to build your apps w
 ## Unity installation script
 After activating the Unity license as usual, add the following script to install the desired version and modules you wish to use. The example below uses Unity Hub CLI commands to install the specified Unity version as well as the Android and iOS Build Support modules.
 
+Mac:
+
 ```yaml
     - name: Install Unity version
       script: | 
         /Applications/Unity\ Hub.app/Contents/MacOS/Unity\ Hub -- --headless install --version $UNITY_VERSION --changeset $UNITY_VERSION_CHANGESET 
         /Applications/Unity\ Hub.app/Contents/MacOS/Unity\ Hub -- --headless install-modules --version $UNITY_VERSION -m ios android 
 ```
 
+Windows:
+
+```yaml
+    - name: Install Unity version
+      script: | 
+        New-Item ".\install-unity.bat" #create an empty batch file
+
+          Set-Content install-unity.bat "`"$env:UNITY_HUB`" -- --headless install -v $env:UNITY_VERSION --changeset $env:UNITY_VERSION_CHANGESET"
+          Add-Content install-unity.bat "`"$env:UNITY_HUB`" -- --headless install-modules --version $env:UNITY_VERSION -m ios android"
+
+          Start-Process -FilePath ".\install-unity.bat" -Wait -NoNewWindow #start executing the batch file
+```
```

Comment:
Please check the indentation is correct here. The first line looks ok, but the others are indented further.